pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -106,6 +106,7 @@ def synchronize(remote_pk, repository_pk):
     Args:
         remote_pk (str): The remote PK.
         repository_pk (str): The repository PK.
+        skip (list): List of RPM types to skip.
```

Comment:
s/RPM/content/

Reply:
We should also document (somewhere) what the various strings are to be used as options, here.  I don't suppose a ```ChoiceField``` could be nested inside the ```ListField``` so that the choices are validated?

Reply:
Serializer can have a 'child' for validating objects inside the list. Will use that for validation.